site/content/docs/5.0/forms/validation.md

```diff
@@ -183,43 +185,43 @@ We recommend using client-side validation, but in case you require server-side v
     <label for="validationServerUsername" class="form-label">Username</label>
     <div class="input-group">
       <span class="input-group-text" id="inputGroupPrepend3">@</span>
-      <input type="text" class="form-control is-invalid" id="validationServerUsername" aria-describedby="inputGroupPrepend3" required>
-      <div class="invalid-feedback">
+      <input type="text" class="form-control is-invalid" id="validationServerUsername" aria-describedby="inputGroupPrepend3 validationServerUsernameFeedback" required>
```

Comment:
Not sure `@` is a proper description, shouldn't we move the id of the prefix to the label?

Reply:
is is a proper additional description in that it makes it obvious what sighted users would perceive. keep in mind that this description is in addition to the preceding `<label>` that also still acts on the `<input>`, so it's announced as something like "Username, @, input"